weblate/addons/tests.py

```diff
@@ -1577,3 +1585,302 @@ def test_json(self) -> None:
 class TasksTest(TestCase):
     def test_cleanup_addon_activity_log(self) -> None:
         cleanup_addon_activity_log()
+
+
+class WebhookAddonsTest(ViewTestCase):
+    """Test for Webhook Addon."""
+
+    addon_configuration = {
```

Comment:
Add type hint might catch some issues, but I don't think it will catch editing it, it would have to be combined with `Final` which is a Python 3.13 feature:

```suggestion
    addon_configuration : ClassVar[dict] = {
```